gtsam/discrete/DecisionTree-inl.h

```diff
@@ -306,7 +306,8 @@ namespace gtsam {
     void dot(std::ostream& os, const LabelFormatter& labelFormatter,
              const ValueFormatter& valueFormatter,
              bool showZero) const override {
-      os << "\"" << this->id() << "\" [shape=circle, label=\"" << label_
+      const std::string label = labelFormatter(label_);
```

Comment:
I've actually contemplated adding very thin wrapper over `AlgebraicDecisionTree<Key>` so we can have sensible defaults.

E.g. `dt.print()` instead of `dt.print("", DefaultKeyFormatter)` for one.

Do you think it's a good idea? I can do it later during the week.

Reply:
Let's focus on estimation instead...

Reply:
Defaults are already in place...